java/src/test/android/README.md

```diff
@@ -0,0 +1,36 @@
+# Android Test Application for ORT-Mobile 
+
+This directory contains a simple android application for testing [ONNX Runtime AAR package](https://www.onnxruntime.ai/docs/how-to/build.html#build-android-archive-aar).
+
+## Background
+
+For general usage and build purpose of ORT-Mobile Android, please see the [documentation](https://www.onnxruntime.ai/docs/how-to/build.html#android) here.
+
+### Test Android Application Overview
+
+This android application is mainly aimed for testing:
+
+- Model used: A simple [sigmoid ONNX model](https://github.com/onnx/onnx-docker/blob/master/onnx-ecosystem/inference_demos/simple_onnxruntime_inference.ipynb) (converted to ORT format under `app\src\androidTest\assets` folder)
```

Comment:
it might be good to link to the exact model file with the version. I guess it's this?
https://github.com/onnx/onnx/blob/f9b0cc99344869c246b8f4011b8586a39841284c/onnx/backend/test/data/node/test_sigmoid/model.onnx
also might be useful to provide the command line invocation for generating the ORT format model, in case we ever need to update it later.